release/config_generator.html

```diff
@@ -0,0 +1,214 @@
+<!doctype html>
```

Comment:
should we upgrade this to buildkite input step?

Reply:
Let's do this in a separate PR